### PR TITLE
New Isolation Migration page

### PR DESCRIPTION
**Pull request by david-richer-adsk**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/47a8341e116f4d7b3c2240d193cc7eac7922dc7b/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/be266f802e0eb6e350552ff7f723a208b9201428/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/3e9972ec3c469487b8280a29b7c29912fd156a4e/index.html)

File: docs/en/trusted-solutions/tier1/setup/migration.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (guillaume-brossard-adsk) [inline docs/en/trusted-solutions/tier1/setup/migration.md:1]: Can you add a "Next Step" section in Fine Tuning pointing to Migration?

File: docs/en/trusted-solutions/tier1/setup/migration.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# Migration
+
+Once everything is configured and properly tested with the migration test site, it's now time to migrate your production site to use the isolation features.
+
+## Test migration
+
+The first step is to do a test migration.
+
```
Comment (guillaume-brossard-adsk) [inline docs/en/trusted-solutions/tier1/setup/migration.md:15]: Ask the Shotgun team to start the migration process in support ticket/slack

**Review by guillaume-brossard-adsk — CHANGES_REQUESTED** [review]

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/23db79cbeb74d128c5e239dee63b9028561d7c54/index.html)

**Comment by david-richer-adsk** [comment]
CR fix done

**Review by guillaume-brossard-adsk — APPROVED** [review]

**Pull request merged**